src/createInterceptor.ts

```diff
@@ -19,6 +19,7 @@ export interface IsomorphicRequest {
   url: URL
   method: string
   headers: Headers
+  credentials: 'omit' | 'include' | 'same-origin'
```

Comment:
I'm preserving the "same-origin" option because the `fetch` interceptor is designed for the browser where such an option is legitimate and possible. It's a no-op in Node.js.